src/index.html

```diff
@@ -5,8 +5,393 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="./images/logo.png" alt="moyo_logo"></img>
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#apple" class="nav__link nav__link--active is-active">apple</a>
+          </li>
+
+          <li>
+            <a href="#samsung"
+            class="nav__link">samsung</a>
+          </li>
+
+          <li>
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+
+          <li>
+            <a data-qa="nav-hover" href="#laptops&computers"
+            class="nav__link">laptops & computers
+            </a>
+          </li>
+
+          <li>
+            <a href="#gadgets" class="nav__link">gadgets</a>
+          </li>
+
+          <li>
+            <a href="#tablets" class="nav__link">tablets</a>
+          </li>
+
+          <li>
+            <a href="#photo" class="nav__link">photo</a>
+          </li>
+
+          <li>
+            <a href="#video" class="nav__link nav__link--last">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container">
+
+        <div class="card" data-qa="card">
+          <img src="./images/imac.jpeg" class="card__img"
+          alt="APPLE iMac 27 inch Retina 5K Monoblock">
+
+          <div class="card__title">
+            <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          </div>
+
+          <div class="card__product">
+            <p>Product code: 195434</p>
+          </div>
+
+          <div class="card__reviews">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__reviews-count">
+              <p>Reviews: 5</p>
+            </div>
+          </div>
+
+          <div class="price">
+            <div class="price__text">
+              <p>Price:</p>
+            </div>
+
+            <div class="price__amount">
+              <p>$2,199</p>
+            </div>
+          </div>
+
+          <button class="button">
+            <a href="index.html" class="button__link"
+            data-qa="card-hover">BUY</a>
+          </button>
```

Comment:
Honestly there is no need for using both `button` and `a` tag. You should choose `a` because the main purpose of this element is to be a link.